src/commonMain/kotlin/exchange.dydx.abacus/output/input/TradeInput.kt

```diff
@@ -678,8 +678,12 @@ enum class OrderType(val rawValue: String) {
     ;
 
     companion object {
-        operator fun invoke(rawValue: String?) =
-            OrderType.values().firstOrNull { it.rawValue == rawValue }
+        operator fun invoke(rawValue: String?): OrderType? {
+            // hacky fix until indexer returns underlying type
+            if (rawValue == liquidation.rawValue) return limit
+            return OrderType.values().firstOrNull { it.rawValue == rawValue }
```

Comment:
I'm bad at kotlin but do we not have to return `OrderType.limit`? (instead of just `limit`?

Reply:
i think because it's within the enum class itself, no we don't